6.3.0/aspnet-core/src/BhResturant.Application/Product/Dto/ProductDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Abp.Application.Services.Dto;
+using Abp.AutoMapper;
+using System.ComponentModel.DataAnnotations;
+
+
+namespace BhResturant.Products.Dto
+{
+    [AutoMapFrom(typeof(Product))]
+    [AutoMapTo(typeof(Product))]
+    public class ProductDto : EntityDto<int>
```

Comment:
I think the use of [MultiLingualEntity](https://aspnetboilerplate.com/Pages/Documents/Multi-Lingual-Entities) will be better here. WDYT?